cloudify-manager-worker/templates/statefulset.yaml

```diff
@@ -21,6 +21,37 @@ spec:
     spec:
       automountServiceAccountToken: false
       securityContext: {{- toYaml .Values.podSecurityContext | nindent 8 }}
+      initContainers:
```

Comment:
Super cool idea with the initContainer that waits for dependencies. Unrelated to this PR but I like the idea of customizing a script in such an environment to have an accurate check - I hope we can do the same for the readiness probe while 'cfy_manager configure' is running